<ui version="4.0" >
 <class>DebugDlg</class>
 <widget class="QDialog" name="DebugDlg" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1005</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Debug</string>
  </property>
  <property name="sizeGripEnabled" >
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QWidget" native="1" name="widget_14" >
     <property name="minimumSize" >
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <layout class="QHBoxLayout" >
      <property name="spacing" >
       <number>10</number>
      </property>
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="frame_5" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="m_pSaveFileInfoB" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip" >
            <string>Save file info</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="Mp3Diags.qrc" >:/images/save_info.svg</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>33</width>
             <height>36</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="m_pSortByShortNamesCkB" >
           <property name="text" >
            <string>Sort by short names</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_8" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" >
         <property name="leftMargin" >
          <number>9</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="m_pEnableTracingCkB" >
           <property name="text" >
            <string>Enable tracing</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_pSaveLogB" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip" >
            <string>Save trace messages</string>
           </property>
           <property name="text" >
            <string>...</string>
           </property>
           <property name="icon" >
            <iconset resource="Mp3Diags.qrc" >:/images/save_log.svg</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>33</width>
             <height>36</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_14" >
        <property name="frameShape" >
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" >
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QLineEdit" name="m_pFrameHdrE" >
           <property name="sizePolicy" >
            <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize" >
            <size>
             <width>80</width>
             <height>16777215</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="m_pDecodeMpegFrameB" >
           <property name="minimumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="maximumSize" >
            <size>
             <width>40</width>
             <height>40</height>
            </size>
           </property>
           <property name="focusPolicy" >
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="toolTip" >
            <string>Decode MPEG Audio frame header</string>
           </property>
           <property name="text" >
            <string>dec</string>
           </property>
           <property name="icon" >
            <iconset resource="Mp3Diags.qrc" >:/images/decode.svg</iconset>
           </property>
           <property name="iconSize" >
            <size>
             <width>36</width>
             <height>36</height>
            </size>
           </property>
           <property name="autoRaise" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="m_pTst01B" >
        <property name="minimumSize" >
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize" >
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
        <property name="focusPolicy" >
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip" >
         <string>Test</string>
        </property>
        <property name="text" >
         <string>tst</string>
        </property>
        <property name="icon" >
         <iconset resource="Mp3Diags.qrc" >:/images/test.svg</iconset>
        </property>
        <property name="iconSize" >
         <size>
          <width>36</width>
          <height>36</height>
         </size>
        </property>
        <property name="autoRaise" >
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>10</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QWidget" native="1" name="widget" >
        <layout class="QHBoxLayout" >
         <property name="leftMargin" >
          <number>0</number>
         </property>
         <property name="topMargin" >
          <number>0</number>
         </property>
         <property name="rightMargin" >
          <number>0</number>
         </property>
         <property name="bottomMargin" >
          <number>0</number>
         </property>
         <item>
          <widget class="QPushButton" name="m_pCloseB" >
           <property name="text" >
            <string>Close</string>
           </property>
           <property name="default" >
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" native="1" name="widget_2" >
     <layout class="QHBoxLayout" >
      <property name="leftMargin" >
       <number>0</number>
      </property>
      <property name="topMargin" >
       <number>0</number>
      </property>
      <property name="rightMargin" >
       <number>0</number>
      </property>
      <property name="bottomMargin" >
       <number>0</number>
      </property>
      <item>
       <widget class="QCheckBox" name="m_pUseAllNotesCkB" >
        <property name="text" >
         <string>Use all notes</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pLogTransfCkB" >
        <property name="text" >
         <string>Log transformations</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="m_pSaveDownloadedDataCkB" >
        <property name="text" >
         <string>Save downloaded data</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_6" >
     <property name="text" >
      <string>Trace messages:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="m_pLogG" />
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Mp3Diags.qrc" />
 </resources>
 <connections/>
</ui>
